### PR TITLE
app angular upgrade core and cli

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/integrations/netsuite/netsuite-shared/netsuite-export-settings/netsuite-export-settings.component.ts [hunk 1]
```diff
@@ -228,10 +228,7 @@ export class NetsuiteExportSettingsComponent implements OnInit {
       this.exportSettingForm.controls.creditCardExportGroup.setValue(ExpenseGroupingFieldOption.EXPENSE_ID);
       this.exportSettingForm.controls.creditCardExportGroup.disable();
     } else {
-
       this.cccExpenseGroupingDateOptions = ExportSettingModel.constructGroupingDateOptions(this.exportSettingForm.value.creditCardExportGroup, NetSuiteExportSettingModel.getReimbursableExpenseGroupingDateOptions());
-      this.helperService.clearValidatorAndResetValue(this.exportSettingForm, 'creditCardExportGroup');
-      this.helperService.enableFormField(this.exportSettingForm, 'creditCardExportGroup');
```
Comment (ashwin1111) [inline src/app/integrations/netsuite/netsuite-shared/netsuite-export-settings/netsuite-export-settings.component.ts:233]: why is this removed?

File: src/app/integrations/qbo/qbo-onboarding/qbo-clone-settings/qbo-clone-settings.component.ts [hunk 2]
```diff
@@ -182,7 +182,7 @@ export class QboCloneSettingsComponent implements OnInit {
 
   resetCloneSetting(): void {
     this.warningHeaderText = 'Are you sure?';
-    this.warningContextText = `By resetting the configuration, you will be configuring each setting individually from the beginning.<br><br>Would you like to continue?`;
+    this.warningContextText = `By resetting the configuration, you will be configuring each setting individually from the beginning.`;
```
Comment (ashwin1111) [inline src/app/integrations/qbo/qbo-onboarding/qbo-clone-settings/qbo-clone-settings.component.ts:185]: why is this removed?

File: src/app/integrations/xero/xero-onboarding/xero-clone-settings/xero-clone-settings.component.ts [hunk 3]
```diff
@@ -149,7 +149,7 @@ export class XeroCloneSettingsComponent implements OnInit {
 
   resetCloneSetting(): void {
     this.warningHeaderText = 'Are you sure?';
-    this.warningContextText = `By resetting the configuration, you will be configuring each setting individually from the beginning.<br><br>Would you like to continue?`;
```
Comment (ashwin1111) [inline src/app/integrations/xero/xero-onboarding/xero-clone-settings/xero-clone-settings.component.ts:152]: why is this removed

File: src/styles.scss [hunk 4]
```diff
@@ -779,3 +779,11 @@ p {
 ::-webkit-scrollbar-thumb:hover {
     @apply tw-bg-[#aaa] tw-rounded-99-px;
 }
+
+body > div:nth-child(3) {
```
Comment (ashwin1111) [inline src/styles.scss:783]: what are we fixing here?

File: src/styles.scss [hunk 5]
```diff
@@ -779,3 +779,11 @@ p {
...
+    @apply tw-fixed #{!important};
+}
+
+body > div > div:nth-child(1) {
```
Comment (ashwin1111) [inline src/styles.scss:787]: what are we fixing here?

File: src/styles.scss (hunk 4 above)
Comment (DhaaraniCIT) [reply src/styles.scss]: this is for the tax wala dropdown ka fix, removed this

Comment (ashwin1111) [reply src/styles.scss]: not a good approach, pls apply to specific element

File: src/styles.scss (hunk 5 above)
Comment (ashwin1111) [reply src/styles.scss]: not a good approach, pls apply to specific element

Comment (DhaaraniCIT) [reply src/styles.scss]: this is removed
